go-apps/meep-rnis/server/logger.go

```diff
@@ -13,159 +13,35 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  *
- * AdvantEDGE RNI Service REST API
+ * AdvantEDGE Location Service REST API
```

Comment:
Should be RNI Service. Remove all references to Location Service in RNI Service code.

Reply:
fixed